frontend/src/components/Result.js

```diff
@@ -111,13 +117,13 @@ export class Result extends React.Component {
         </div>
 
         <div className="container-plattforms">
-          <div classNme="column-one-sixed"></div>
+          <div className="column-one-sixed"></div>
```

Comment:
```suggestion
          <div className="column-one-sixed" />
```
In general, I would advice you, to install [Prettier](https://prettier.io/), that does such things automatically and ensures a uniform codestyle.
In combination with [Pretty Quick](https://github.com/azz/pretty-quick) it runs as a pre-commit hook over the changed code, so you don't have to do it manually each time.

Reply:
I installed it already, but didn't activated it for the project. And I am also using Pretty Quick now. Thanks for the advise!